package.json

```diff
@@ -17,13 +17,16 @@
     "@types/node": "20.6.3",
     "@types/react": "18.2.22",
     "@types/react-dom": "18.2.7",
+    "country-state-city": "^3.2.1",
```

Comment:
will ask ambreen if we can limit to U.S. cities for now

Reply:
if not, we should consider fields for `country` and `state`/`province`